docs/sources/tutorials/alerting-get-started/alerting-get-started.md

```diff
@@ -0,0 +1,144 @@
+# Alerting: Get started
+
+In this guide, we'll walk you through the process of setting up your first alert in just a few minutes. You'll witness your alert in action with real-time data, as well as receiving alert notifications.
+
+## Requirements
+
+Docker Compose (included in Docker for Desktop for macOS and Windows)
```

Comment:
This probably needs bullet points, currently renders the items on the same line as a single sentence and looks a bit odd :)